### PR TITLE
readline fix and exit

### PR DESCRIPTION
**Pull request by JackBoreczky**
Add an import for pyreadline if readline errors (happens on windows). Also add another way to exit the game.

### CODE REVIEW

File: eat_shovel.py [hunk 1]
```diff
@@ -61,11 +64,16 @@ def play(self):
...
+def exit_game(player, _cmd):
+    raise EOFError
+
 def main():
     # everything you run on startup is in here (unless its global init)
     start_room = Room()
     cmd_map = {
-        "nuke": act_nuke
+        "nuke": act_nuke,
+        "exit": exit_game
```
Comment (bitbloxhub) [inline eat_shovel.py:76]: add trailing comma

Comment (bitbloxhub) [inline eat_shovel.py:76]: add trailing comma

**Review by bitbloxhub — CHANGES_REQUESTED** [review]
nice! but see below

File: eat_shovel.py (hunk 1 above)
Comment (JackBoreczky) [reply eat_shovel.py]: resolved!

**Review by bitbloxhub — APPROVED** [review]
ok

**Comment by bitbloxhub** [comment]
do not delete branch

**Pull request merged**